lib/open_mic.rb

```diff
@@ -0,0 +1,28 @@
+class OpenMic
+	attr_reader :location, :date, :performers
+	
+	def initialize(hash)
+		@location = hash[:location]
+		@date = hash[:date]
+		@performers = []
+	end
+
+	def welcome(user)
+		@performers << user 
+	end
+
+	def repeated_jokes?
+		jokes = []
+		@performers.each do |user|
+			user.jokes.each do |joke|
+				jokes << joke
+			end
+		end
+		if jokes.count == jokes.uniq.count
+			return false
+		else
+			return true
+		end
+	require 'pry'; binding.pry
```

Comment:
Good job getting this to work.  This could also be accomplished with a lot less code by doing: 

```
  def repeated_jokes?
    all_jokes = @performers.map do |user|
      user.jokes
    end

    all_jokes.flatten.length != all_jokes.flatten.uniq.length
  end
```

The logic is fundamentally the same, but uses tools/syntax more efficiently.